manuscript/teaching-reproducibility.Rmd

```diff
@@ -133,6 +133,35 @@ that are particularly effective for teaching reproducibility successfully:
 In this paper, we will discuss why we believe each of these are important,
 provide examples of how to incorporate these in your teaching.
 
+## Academic setting and background
+
+The specific teaching pedagogies we present in this paper
+have been tested in classrooms of 10-200 learners from varying educational backgrounds,
+including undergraduate students taking their only data science course
+and graduate students looking to specialize in data science.
+We teach the plurality of our courses in the Master of Data Science program,
+where students are generally required to have taken one course each
+in programming, calculus, linear algebra and introductory statistics and probability
+before entering the program.
```

Comment:
This is redundant to what I added above... So we somehow need to blend these two. I feel like we should now mention that the University we teach at is the University of British Columbia. And that the undergraduate course is DSCI 100 (Introduction to Data Science) and the graduate program is a professional Master of Data Science program, consisting of 24 one-credit courses, culminating in and a six-credit capstone course.

Reply:
We do say we teach at UBC in the beginning of the intro. I added the additional info about MDS, but not the undergrad course name since I don't think it adds much over saying that we teach an introductory undergrad course in DS (and there is also 310).